graphql/schema.graphql

```diff
@@ -1,5 +1,12 @@
 type Query {
-  userPatches(userId: String!): [Patch!]!
+  userPatches(
+    limit: Int = 0
+    page: Int = 0
+    patchName: String = ""
+    statuses: [String!] = []
+    userId: String
```

Comment:
why is userId included as query param when user is obtained in resolver via `route.MustHaveUser(ctx)` ?

Reply:
it's an optional param that you can provide to get patches for a specified user